src/test/ts/browser/InitTest.ts

```diff
@@ -1,83 +1,86 @@
+        Assertions.assertEq('Editor should not be inline', false, vmContext.editor.inline);
+      });
 
-      Logger.t('Test one way binding tinymce-vue -> variable', GeneralSteps.sequence([
-        Logger.t('Test outputFormat="text"', Chain.asStep({}, [
-          cRender({
-            content: undefined
-          }, `
-            <editor
-              :init="init"
-              @update:modelValue="content = $event"
-              output-format="text"
-            ></editor>
-          `),
-          cFakeType('A'),
-          Chain.op((context) => {
-            Assertions.assertEq('Content emitted should be of format="text"', 'A', context.vm.content);
-          }),
-          cRemove
-        ])),
-        Logger.t('Test outputFormat="html"', Chain.asStep({}, [
-          cRender({
-            content: undefined
-          }, `
-            <editor
-              :init="init"
-              v-model="content"
-              output-format="html"
-            ></editor>
-          `),
-          cFakeType('A'),
-          Chain.op((context) => {
-            Assertions.assertEq('Content emitted should be of format="html"', '<p>A</p>', context.vm.content);
-          }),
-          cRemove
-        ])),
-      ])),
-    ])
-  );
+      it('should be inline with inline attribute in template', async () => {
+        const vmContext = await pRender({}, `
+          <editor
+            :init="init"
+            :inline="true"
+          ></editor>`);
+        Assertions.assertEq('Editor should be inline', true, vmContext.editor.inline);
+      });
 
-  Pipeline.async({}, [
-    sTestVersion('4'),
-    sTestVersion('5'),
-    sTestVersion('6'),
-    sTestVersion('7'),
-  ], success, failure);
+      it('should be inline with inline option in init', async () => {
+        const vmContext = await pRender({ init: { inline: true }});
+        Assertions.assertEq('Editor should be inline', true, vmContext.editor.inline);
+      });
+
+      it('should handle one-way binding with output-format="text"', async () => {
+        const vmContext = await pRender({
+          content: undefined,
+        }, `
+          <editor
+            :init="init"
+            api-key="${VALID_API_KEY}"
+            @update:modelValue="content=$event"
+            output-format="text"
+          ></editor>
+        `);
+        await pFakeType('A', vmContext);
+        await Waiter.pWait(100);
+        Assertions.assertEq('Content emitted should be of format="text"', 'A', vmContext.vm.content);
+      });
+
+      it('should handle one-way binding with output-format="html"', async () => {
+        const vmContext = await pRender({
+          content: undefined,
+        }, `
+          <editor
+            :init="init"
+            api-key="${VALID_API_KEY}"
+            @update:modelValue="content=$event"
+            output-format="html"
+          ></editor>
+        `);
+        await pFakeType('A', vmContext);
+        await Waiter.pWait(100);
```

Comment:
```suggestion
```

Reply:
Not required, removed in [this commit](https://github.com/tinymce/tinymce-vue/pull/434/commits/6bb5d8f1f41a7d0458ed7889335c20a06de33c71)